pkg/azure/client.go

```diff
@@ -322,7 +322,7 @@ func (az *azClient) DeployGatewayWithSubnet(subnetID, skuName string) (err error
 		return
 	}
 	if result.Name != nil {
-		klog.Infof("Completed deployment %v: %v", deploymentName, *result.Properties.ProvisioningState)
+		klog.Infof("Completed deployment %v: %v", deploymentName, result.Properties.ProvisioningState)
```

Comment:
is it because of the sdk version change?

Reply:
Yes